### DISCUSSION TITLE
Error in installing package

### DISCUSSION THREAD

**Discussion by LoyceMajige** [IBAHCM/RPiR#296] [Archive]
Hi @soniamitchell , @richardreeve,

While installing one of my group member's package so i can check the demo this is the error i am getting, may you please help me on what might be the problem?

Running "C:/PROGRA~1/R/R-40~1.3/bin/x64/Rcmd.exe" INSTALL \
  "C:\Users\LOYCEM~1\AppData\Local\Temp\Rtmpy2wsgN/NambungaBCI_0.1.1.tar.gz" --install-tests 
* installing to library 'C:/Users/loyce majige/Documents/R/win-library/4.0'
ERROR: dependency 'NambungaBCI' is not available for package 'NambungaBCI'
* removing 'C:/Users/loyce majige/Documents/R/win-library/4.0/NambungaBCI'
Error in (function (command = NULL, args = character(), error_on_status = TRUE,  : 
  System command 'Rcmd.exe' failed, exit status: 1, stdout & stderr were printed
Type .Last.error.trace to see where the error occured
In addition: There were 21 warnings (use warnings() to see them)

Thank you!
Loyce:)
>

**Comment by soniamitchell** [comment]
Hi @LoyceMajige, NambungaBCI has a dependency on itself (which is wrong). Could you ask Ismail to remove NambungaBCI from the Imports field in the DESCRIPTION file (also to remove the comma after magrittr).

**Reply by richardreeve** [reply]
@LoyceMajige - note that I've moved this message to the discussion here as it wasn't an issue with the RPiR package.
